google/cloud/bigquery/table.py

```diff
@@ -1690,6 +1728,8 @@ def to_arrow(
         """
         self._maybe_warn_max_results(bqstorage_client)
 
+        self._maybe_warn_max_results(bqstorage_client)
+
```

Comment:
IIRC I fixed that recently, but it might have been in the `v3` branch. Anyhow, the duplicate call should be removed.
```suggestion
```

Hm... but I can't seem to suggest a completely empty replacement, thus we'll end up with two blank lines and the linter will likely complain... manual change on the PR branch might be needed.

Reply:
Good eye! Just pushed a fix.